scripts/qa_crowbarsetup.sh

```diff
@@ -3577,10 +3587,7 @@ function oncontroller_run_tempest()
     fi
     local imageid=$(glance_image_get_id $image_name)
     crudini --set /etc/tempest/tempest.conf orchestration image_ref $imageid
-    # test if is cnftools image prepared for tempest
-    wait_for 300 5 \
-        'openstack image show $imageid | grep active &>/dev/null' \
-        "prepare cnftools image"
+    wait_image_active "$image_name" tempest
```

Comment:
and now you're showing $image_name, which might not necessarily work.. can we add imageid as a 1st parameter and use the name as a 2nd? I am not sure the "purpose" parameter is really needed.